src/index.css

```diff
@@ -11,3 +11,8 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
     monospace;
 }
+
```

Comment:
lets use CSS Grid instead of flex it is the next step in responsive web design and very straightforward.